test/zepto.html

```diff
@@ -2175,6 +2175,14 @@
           $.map(el.children(), function(c){ return $(c).text() }).join(' '))
       },
 
+      testAppendWithArrayParams: function(t){
+        var node = document.createElement('b');
+        node.appendChild(document.createTextNode('four'))
+        var el = $('<div></div>')
+        el.append(['<b>one</b><b>two</b>', '<b>three</b>', node])
+        t.assertEqual('<b>one</b><b>two</b><b>three</b><b>four</b>', el.html())
```

Comment:
Test looks good. Can you please include another test that verifies that one of the elements in the array can also be a Zepto collection that contains for example 2 items?